test_scripts/WebEngine/WebEngineProjection/024_OnExitApplication_RESOURCE_CONSTRAINT_multiple_apps_on_one_connection.lua

```diff
@@ -0,0 +1,39 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Processing of the OnExitApplication notification with reason RESOURCE_CONSTRAINT from HMI
+--  (multiple applications on one connection)
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App1 and App2 are registered on one mobile connection
+--
+-- Steps:
+-- 1. HMI sends BC.OnExitApplication with reason: "RESOURCE_CONSTRAINT" related to App2 to SDL
+-- SDL does:
+--  - unregister App2 and send OnAppInterfaceUnregistered notification with reason: "RESOURCE_CONSTRAINT" to it
+--  - send BasicCommunication.OnAppUnregistered notification related to App2 with unexpectedDisconnect: false
+--  - does not close mobile connection
+--  - does not send send OnAppInterfaceUnregistered notification with reason: "RESOURCE_CONSTRAINT" to App1
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+-- local events = require("events")
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local APP_1 = 1
+local APP_2 = 2
```

Comment:
```suggestion
local app1_id = 1
local app2_id = 2
```

Reply:
@YarikMamykin @aderiabin The same here? https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/34/files#diff-68a100323919bd7a65d241ecaa97c733R31-R32 
For me it's doesn't matter if it would be in upper case or no. let's just keep consistency

Reply:
@YarikMamykin All variables/constants is renamed using `camelCase` style (see d8172a5 )